@pujaoshahu  build errors

```
./src/app/ceph/cephfs/cephfs.module.ts:27:0-103 - Error: Module not found: Error: Can't resolve './cephfs-mirroring-list/cephfs-mirroring-list.component' in '/home/jenkins-build/build/workspace/ceph-api/src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs'

./src/app/ceph/cephfs/cephfs.module.ts:28:0-106 - Error: Module not found: Error: Can't resolve './cephfs-mirroring-error/cephfs-mirroring-error.component' in '/home/jenkins-build/build/workspace/ceph-api/src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs'

./src/app/ceph/cephfs/cephfs.module.ts:34:0-109 - Error: Module not found: Error: Can't resolve './cephfs-mirroring-wizard/cephfs-mirroring-wizard.component' in '/home/jenkins-build/build/workspace/ceph-api/src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs'

./src/app/ceph/cephfs/cephfs.module.ts:35:0-118 - Error: Module not found: Error: Can't resolve './cephfs-filesystem-selector/cephfs-filesystem-selector.component' in '/home/jenkins-build/build/workspace/ceph-api/src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs'

./src/app/ceph/cephfs/cephfs.module.ts:36:0-109 - Error: Module not found: Error: Can't resolve './cephfs-mirroring-entity/cephfs-mirroring-entity.component' in '/home/jenkins-build/build/workspace/ceph-api/src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs'

Error: src/app/app.module.ts:32:5 - error NG6002: This import contains errors, which may affect components that depend on this NgModule.

32     CephModule
       ~~~~~~~~~~


Error: src/app/ceph/ceph.module.ts:18:5 - error NG6002: 'CephfsModule' does not appear to be an NgModule class.

18     CephfsModule,
       ~~~~~~~~~~~~
....

```

--- a/src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs.module.ts
+++ b/src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs.module.ts
@@ -46,6 +46,7 @@ import {
   NumberModule,
   PlaceholderModule,
   SelectModule,
+  TagModule,
   TimePickerModule,
   TreeviewModule,
   TabsModule
@@ -86,7 +87,8 @@ import Trash from '@carbon/icons/es/trash-can/32';
     ComboBoxModule,
     IconModule,
     BaseChartDirective,
-    TabsModule
+    TabsModule,
+    TagModule
   ],
   declarations: [
     CephfsDetailComponent,

